czo2hs.py

```diff
@@ -157,17 +158,17 @@ def migrate_czo_row(czo_row_dict, row_no=1, progress_report=5):
                                                                        error_item["error_msg_list"]])))
     logging.info(text_emphasis("CZO_ID <---> HS_ID Lookup Table"))
     logging.info(czo_hs_id_lookup_df.to_string())
-    results_file = os.path.join(logdir, 'czo_hs_id_{}.csv'.format(start_time))
-    logging.info("*" * 20 + "Saving Lookup Table to {}".format(results_file) + "*" * 20)
+    czo_hs_id_lookup_csv_path = os.path.join(logdir, 'czo_hs_id_{}.csv'.format(start_time_str))
+    logging.info(text_emphasis("Saving Lookup Table to {}".format(czo_hs_id_lookup_csv_path)))
 
-    czo_hs_id_lookup_df.to_csv(results_file, encoding='utf-8', index=False)
+    czo_hs_id_lookup_df.to_csv(czo_hs_id_lookup_csv_path, encoding='utf-8', index=False)
 
     # upload log file and results file to hydroshare
-    logging.info("*"*20 + "Uploading log file to HS" + "*"*20)
+    logging.info(text_emphasis("Uploading log file to HS"))
     hs = CZO_HS_Account_Obj.get_hs_by_czo("default")
     hs_id = hs.createResource("CompositeResource",
-                              "czo2hs migration log files {}".format(start_time),
+                              "czo2hs migration log files {}".format(start_time_str),
                               )
```

Comment:
don't create a new variable for start_time_str, just str(start_time) here in line 170

Reply:
don't include data types in variable names. rename this from start_time_str to start_time

Reply:
fixed in new commit. 
I want to specify the formatting of the datetime str as it is used in filename which should not contain ":" or spaces